p2p/memp2p/memp2p.go

```diff
@@ -334,7 +334,7 @@ func (messenger *Messenger) SendToConnectedPeer(topic string, buff []byte, peerI
 // previously registered a message processor for that topic. The Network will
 // log the message only if the Network.LogMessages flag is set and only if the
 // Messenger has the requested topic and MessageProcessor.
-func (messenger *Messenger) ReceiveMessage(topic string, message p2p.MessageP2P) error {
+func (messenger *Messenger) ReceiveMessage(topic string, message p2p.MessageP2P, allowBroadcast bool) error {
```

Comment:
broadcast is enough for parameter name.

Reply:
I would keep this allow part as to be more clear of what "broadcast" should mean.